src/components/designSystem/Drawer.tsx

```diff
@@ -119,7 +129,8 @@ const Header = styled.div`
   }
 `
 
-const Content = styled.div`
+const Content = styled.div<{ $fullContentHeight?: boolean }>`
+  height: ${({ $fullContentHeight }) => ($fullContentHeight ? '100%' : ' ')};
```

Comment:
@ansmonjol Not always, when the content inside the drawer is longer than the viewport height, we don't want to set the height to 100% otherwise it's breaking the container padding-bottom